packages/tupaia-web-server/src/routes/CountryAccessListRoute.ts

```diff
@@ -17,6 +17,6 @@ export type CountryAccessListRequest = Request<
 export class CountryAccessListRoute extends Route<CountryAccessListRequest> {
   public async buildResponse() {
     const { ctx } = this.req;
-    return ctx.services.central.fetchResources('me/countries');
+    return ctx.services.central.fetchResources('me/countries', { disableAdmin: true });
```

Comment:
Extremely WET for no reason. But the current boilerplate code makes it difficult to override some these connections in a useful way. I've split out a ticket WAITP-1433 to make that easier and get rid of all these hard set disables.